docs/cmdline-opts/write-out.d

```diff
@@ -39,6 +39,9 @@ option. (Added in 7.26.0)
 The initial path curl ended up in when logging on to the remote FTP
 server. (Added in 7.15.4)
 .TP
+.B header_count
+The count of retrieved headers.
```

Comment:
I suggest:: "The number of response headers in the most recent request (restarted at each redirect)"

Should this mention that it doesn't include the first status line?

Reply:
> Should this mention that it doesn't include the first status line?

I think it's not necessary  when we are talking about headers.

Reply:
I've just learned that some users think of that as a (special) header too.

Reply:
Oops!
By mentioning this, we also have to mention: the status line which includes the status code and reason message IS NOT a header.

So, do you prefer to mention?

Reply:
I'd prefer to mention it because, as @bagder said, some treat the status line as header line (why???).
If we can be clear on the subject, that we should be.

Reply:
OK. So, @bagder, what is your opinion?

Reply:
Being clear is good, which is why I brought this up to begin with.

Reply:
OK, please give me a good sentence to add.

Reply:
Maybe like:
"The number of response headers in the most recent request (restarted at each redirect). Note that the status line IS NOT a header."

Reply:
Thanks! Added.